Source/WebCore/css/PropertySetCSSStyleDeclaration.cpp

```diff
@@ -238,10 +238,7 @@ String PropertySetCSSStyleDeclaration::getPropertyShorthand(const String& proper
 
 bool PropertySetCSSStyleDeclaration::isPropertyImplicit(const String& propertyName)
 {
-    CSSPropertyID propertyID = cssPropertyID(propertyName);
-    if (!isExposed(propertyID))
-        return false;
-    return m_propertySet->isPropertyImplicit(propertyID);
+    return m_propertySet->isPropertyImplicit(cssPropertyID(propertyName));
 }
```

Comment:
I don't see any other usages of this method in the codebase. Can we remove it altogether?

Reply:
Oh nevermind, it is used: https://searchfox.org/wubkat/search?q=symbol:_ZN7WebCore19CSSStyleDeclaration18isPropertyImplicitERKN3WTF6StringE&redirect=false

Reply:
This function is still used. The uses are in these functions:
```
InspectorStyle::shorthandValue
InspectorStyle::styleWithProperties
webkit_dom_css_style_declaration_is_property_implicit
-[DOMCSSStyleDeclaration isPropertyImplicit:]
```
If someone takes care of all of these, then we can remove the function altogether. I don’t know the inspector well enough to know exactly how to remove the first two. The others could likely just be changed to always return `false`.

Reply:
Looking at the remaining usages, I'm tempted to think it's ok to remove them.

One is to skip implicit props from showing in the WI UI, the other one is GTK API. Given how we use implicit properties nowadays, I'm not sure it still makes sense to skip them. cc @patrickangle 

Anyway, this doesn't need to be done here.

Reply:
Presumably someone has to change the inspector to not rely on knowing a property was set by a shorthand with this flag, deducing it in some other way.

Reply:
Once we remove the function we can also remove the flag, so it will be a nice cleanup!